src/Pipfile

```diff
@@ -51,6 +51,6 @@ django-webtest = "*"
 types-cachetools = "*"
 boto3-mocking = "*"
 boto3-stubs = "*"
-django-model2puml = "*"
+django-model2puml = "0.5.0"
```

Comment:
An alternative to setting this to a version before the bug might have been to set `skip-choices-fields` to True to bypass the [bug](https://github.com/sen-den/django-model2puml/issues/10) in the latest version of this library, but this seemed cleaner and I don't think we are missing anything essential by setting it to an earlier version.